### PR TITLE
Fixes for LaTeX/html export of standard library in coqdoc

### PR DESCRIPTION
**Pull request by herbelin**
**Kind:** bug fix

This partially fixes the compilation of the stdlib in pdf format:
- we slightly constrain the rule for emphasis in coqdoc: a `_` following a `"`, `'` or `` ` `` is not considered to be starting emphasis
- this allows to support e.g. `"_CoqProject"` in a comment (as it happened in `ExtrOCamlFloats.v`)
- we add package `amssymb` in the LaTeX prelude to support the new notation for apartness ≶ in `Reals`

There remain problems: some files use Unicode subscript which are told to be turned into `$_...$` in LaTeX. But doing this would require to write a translation for all possible unicode subscripts, which may require some time.

- [ ] Added / updated test-suite
- [X] Corresponding documentation was added / updated
- [ ] Entry added in the changelog (to do?)

Putting 8.12 because the small change for detecting emphasis in coqdoc is maybe too much.

@Zimmi48: I added a label for `coqdoc` as it seems enough specific. I hope it is ok (or will it break too much the usage?).

### CODE REVIEW

**Comment by Zimmi48** [comment]
> @Zimmi48: I added a label for `coqdoc` as it seems enough specific. I hope it is ok (or will it break too much the usage?).

You did well. These labels are meant to be extended to fill the needs of developers.

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
Needs an assignee from the @coq/coqdoc-maintainers team.

**Comment by herbelin** [comment]
> Needs an assignee from the @coq/coqdoc-maintainers team.

@zimmi48, since it is about documentation, maybe could you be the assignee?

**Comment by Zimmi48** [comment]
I guess I can even though I don't know anything about coqdoc.
I'll leave this open one more day before merging.

**Pull request merged**
